lib/ama_layout/version.rb

```diff
@@ -1,3 +1,3 @@
 module AmaLayout
-  VERSION = '6.9.0.pre'
+  VERSION = '6.9.1.pre'
```

Comment:
would you consider this a bug fix, or a minor bump?

Bug fix would be 6.9.1.pre, minor would be 6.10.0.pre.